### PR TITLE
RoboCup 2019 firmware changes

### PR DESCRIPTION
**Pull request by MathewMacDougall**
### Description
Changes from RoboCup 2019. Radio channel changes and slightly speeding up the robot during Move.

### Testing Done
Validated on field.

### Resolved Issues
None

### Length Justification
None

### CODE REVIEW

**Review by henryymliu — APPROVED** [review]

File: src/thunderbots/firmware/main/main.c [hunk 1]
```diff
@@ -338,10 +338,10 @@ static void run_normal(void) {
...
 		uint16_t pan;
 	} mrf_profiles[4] = {
-		{ 23, false, 0x1846U },
-		{ 23, false, 0x1847U },
-		{ 23, false, 0x1848U },
-		{ 23, false, 0x1849U },
+		{ 24, false, 0x1846U },
+		{ 24, false, 0x1847U },
+		{ 24, false, 0x1848U },
+		{ 24, false, 0x1849U },
```
Comment (garethellis0) [inline src/thunderbots/firmware/main/main.c:344]: I gotta ask, why?

**Review by garethellis0 — APPROVED** [review]
LGTM :+1: :tada:

File: src/thunderbots/firmware/main/main.c (hunk 1 above)
Comment (henryymliu) [reply src/thunderbots/firmware/main/main.c:344]: We were assigned this channel at competition.

Comment (garethellis0) [reply src/thunderbots/firmware/main/main.c:344]: Just as a thought, perhaps we could make this a constant?

**Pull request merged**